plutus-tx-plugin/test/Plugin/Basic/patternMatchDo.plc.golden

```diff
@@ -9,6 +9,11 @@
         (vardecl Just (fun a [ Maybe a ])) (vardecl Nothing [ Maybe a ])
       )
     )
+    (termbind
+      (strict)
+      (vardecl fApplicativeMaybe_cpure (all a (type) (fun a [ Maybe a ])))
```

Comment:
The reason for this additional `termBind` is because the `CoreExpr` in GHC 9 contains `$fApplicativeMaybe_$cpure`, while in GHC 8 it is `Just`. In other words, in GHC 9 `$fApplicativeMaybe_$cpure` is not inlined. I _think_ in GHC 8 this is an unconditional pre-inline, but probably some new features (levity polymorphism?) in GHC 9 interferes with it.